content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,132 @@
+:menuselection:`IoT app --> IoT Boxes` and clicking on the :guilabel:`IP address` of the :abbr:`IoT
+(Internet of Things)` box.
+
+Next, click :guilabel:`Handlers list` and then select :guilabel:`Load Handlers` at the bottom of the
+page.
+
+.. image:: updating_iot/load-handlers.png
+   :align: center
+   :alt: Handlers list on an IoT box with the load handlers button highlighted.
+
+.. seealso::
+   Handlers code is fetched from the configured server and it needs to be up-to-date to have the
+   latest fixes and patches.
+
+.. note::
+   The handlers update is also performed automatically each time the :abbr:`IoT (Internet of
+   Things)` box is restarted. The only exception to this process is if the :guilabel:`Automatic
+   drivers update` is unchecked in the form view of the :abbr:`IoT (Internet of Things)` box on the
+   Odoo server. This setting can be reached by going to :menuselection:`IoT App --> Select the IoT
+   box --> Automatic drivers update`.
+
+Upgrade from the IoT box home page
+==================================
+
+In the background, the :abbr:`IoT (Internet of Things)` box uses a version of Odoo code to run and
+connect to the Odoo database. This code may need to be updated in order for the :abbr:`IoT (Internet
+of Things)` box to operate effectively. This operation should be completed on a routine basis, to
+ensure up-to-date :abbr:`IoT (Internet of Things)` processes and core system.
+
+Go to the :abbr:`IoT (Internet of Things)` box homepage by navigating to :menuselection:`IoT app -->
+IoT Boxes` and clicking on the :guilabel:`IP address` of the :abbr:`IoT (Internet of Things)` box.
+Then click on :guilabel:`Update` (next to the version number).
+
+If a new version of the :abbr:`IoT (Internet of Things)` Box image is available, an
+:guilabel:`Upgrade to _xx.xx_` button will appear at the bottom of the page. Click this button to
+upgrade the unit and the :abbr:`IoT (Internet of Things)` box will then flash itself to the new
+version. All of the previous configurations will be saved.
+
+.. note::
+   This process can take more than 30 minutes. Do not turn off or unplug the :abbr:`IoT (Internet of
+   Things)` box as it would leave it in an inconsistent state. This means that the :abbr:`IoT
+   (Internet of Things)` box will need to be re-flashed with a new image.
+
+   .. seealso::
+      :ref:`flash_sdcard/etcher`.
+
+.. important::
+   An :guilabel:`Upgrade` button will appear if the :abbr:`IoT (Internet of Things)` box is on the
+   latest image. In this case, the :abbr:`IoT (Internet of Things)` box will instead update the core
+   code (without changing the image itself. In other words this button forces Odoo based :abbr:`IoT
+   (Internet of Things)` box code to update with the latest version from GitHub. This can be done
+   **without** an Odoo database to synchronize with.
+
+.. image:: updating_iot/flash-upgrade.png
+   :align: center
+   :alt: IoT box software upgrade in the IoT Box Home Page.
```

Comment:
the content is correct but I find it complicated to follow personally. Maybe try to rethink this part to make it more clear. 
Maybe by precising that the Upgrade button will show only if they are on the latest image and that they need to reflash the card on the latest image first if they want to perform this update